docs/sources/installation/tanka.md

```diff
@@ -35,21 +35,12 @@ jb install github.com/grafana/loki/production/ksonnet/loki@main
 jb install github.com/grafana/loki/production/ksonnet/promtail@main
 ```
 
-Then you'll need to install a kubernetes library:
-
-```bash
-jb install github.com/jsonnet-libs/k8s-alpha/1.16
-```
-
-Next, override the `lib/k.libsonnet` with the following
-
-```jsonnet
-import 'github.com/jsonnet-libs/k8s-alpha/1.16/main.libsonnet'
-```
-
-Be sure to replace the username, password, and the relevant `htpasswd` contents.
-Making sure to set the value for username, password, and `htpasswd` properly,
-replace the contents of `environments/loki/main.jsonnet` with:
+Next, replace the contents of `environments/loki/main.jsonnet` with the YAML below, making sure to replace or modify the following:
+1. Update the `username`, `password`, and the relevant `htpasswd` contents
+1. Update the s3 or gcs variables and `boltdb_shipper_shared_store` variable based on your choice of object storage backend. See here for the full [storage_config](https://grafana.com/docs/loki/latest/configuration/#storage_config). 
+1. Remove the object storage variables that are not relevant for your setup (e.g., remove the s3 variables if you're using gcs)
+1. Update the `container_root_path` value so it reflects your own data root for the Docker Daemon. Run `docker info | grep "Root Dir"` to get the root path.
+1. Update the `schema_config` section so that `from` has today's date and `object-store` has your choice of object storage backend (e.g., s3, gcs)
```

Comment:
If someone wants to send older logs to Loki, then with today's date, they might lose them because Loki would just ignore those logs. So I think we should either explicitly mention it or skip suggesting to set it to today's date.
What do you think?

Reply:
Our default config is to accept 7 days older logs, so maybe we can suggest using it as a reference, but some users would get confused if they want to add a new schema with a new date. So we would need a dedicated section talking about adding a new schema config.

Reply:
Got it. I'll set the docs to suggest "use current date minus 14 days" and then mention that this will need to be modified if they want to have multiple schema configs. I'm primarily trying to target this to someone who is setting up loki from scratch for the first time, so I'll call that out as well. If you're doing that, I would expect you to only start with 1 schema config.